### DISCUSSION TITLE
Cards to the homepage

### DISCUSSION THREAD

**Discussion by ekrmy** [Dem13n/discussion-cards#5] [Ideas]
Hello, I really like this plugin, I would love to do this. The tag I will choose will show the latest posts on the homepage. Can we do that? Or would you consider it in future versions?

![image](https://user-images.githubusercontent.com/79015318/107854586-db7e4000-6e2d-11eb-968b-6b30bda78d17.png)
![image](https://user-images.githubusercontent.com/79015318/107854592-e2a54e00-6e2d-11eb-8952-8e4f99a903cb.png)

good work.

**Comment by Dem13n** [comment]
this should be an extension with a completely different logic, I thought of doing something similar but not now
